truss/templates/server/common/truss_server.py

```diff
@@ -142,7 +142,7 @@ async def predict(
 
         # In the case that the model returns a Generator object, return a
         # StreamingResponse instead.
-        if isinstance(response, AsyncGenerator):
+        if isinstance(response, (AsyncGenerator, Generator)):
```

Comment:
since `postprocess` can return a `Generator`, we need to check for that also here